README.md

```diff
@@ -1,26 +1,119 @@
-<div align="center">
-    
-<h1>LetsBuildTogether Community (LBT) </h1>
+<img src="https://avatars.githubusercontent.com/u/102895308?s=200&v=4" width="50" alt="lbt_logo"/>
```

Comment:
I think it would be more appropriate if we link the logo from the assets directory of this repository?